var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -446,6 +448,13 @@ def cmake_args(self):
         # This disables Legion's CMake build system's logic for targeting the native
         # CPU architecture in favor of Spack-provided compiler flags
         options.append("-DBUILD_MARCH:STRING=")
+
+        if "+openmp" in spec and "+sysomp" in spec:
+            options.append("-DLegion_OpenMP_SYSTEM_RUNTIME=ON")
+
+        if "+gc" in spec:
+            options.append("-DCMAKE_CXX_FLAGS=-DLEGION_GC")
+
```

Comment:
this may require too many changes for this PR, but I believe `in spec` is no longer suggested for use

https://github.com/spack/spack/pull/44348#discussion_r1612192317

Reply:
I'm happy to replace all the uses of `in spec` with `spec.satisfies(...)`. Should I open another PR just for that, or add another commit to this PR? I don't know what Spack expects for a series of small changes to packages. This single PR could conceivably already be made into three PRs, even without the additional proposed change.